src/Styleguide/Pages/Artist/Routes/AuctionResults/index.tsx

```diff
@@ -1,46 +1,231 @@
+import { AuctionResults_artist } from "__generated__/AuctionResults_artist.graphql"
+import { ContextConsumer, ContextProps } from "Components/Artsy"
 import React from "react"
-import { Pagination } from "Styleguide/Components/Pagination"
-import { Box } from "Styleguide/Elements/Box"
-import { Flex } from "Styleguide/Elements/Flex"
-import { Col, Row } from "Styleguide/Elements/Grid"
-import { Separator } from "Styleguide/Elements/Separator"
-import { paginationProps } from "Styleguide/Pages/Fixtures/Pagination"
-import { AuctionItem } from "./AuctionItem"
-import { TableColumns } from "./TableColumns"
-import { TableSidebar } from "./TableSidebar"
-
-export const AuctionResults = () => {
-  const { cursor, callbacks } = paginationProps
-
-  return (
-    <React.Fragment>
-      <Row>
-        <TableSidebar />
-
-        <Col sm={10}>
-          <Row>
-            <TableColumns />
-          </Row>
-
-          <Box pt={0.5}>
-            <Separator />
-          </Box>
-
-          <AuctionItem />
-          <AuctionItem />
-          <AuctionItem />
-          <AuctionItem />
-          <AuctionItem />
-        </Col>
-      </Row>
-
-      <Row>
-        <Col>
-          <Flex justifyContent="flex-end">
-            <Pagination {...cursor} {...callbacks} />
-          </Flex>
-        </Col>
-      </Row>
-    </React.Fragment>
-  )
+import { Subscribe } from "unstated"
+import { AuctionResults } from "./AuctionResults"
+import { FilterState } from "./state"
+
+import {
+  createRefetchContainer,
+  graphql,
+  QueryRenderer,
+  RelayRefetchProp,
+} from "react-relay"
+
+const PAGE_SIZE = 10
+
+interface Props extends ContextProps {
+  artistID: string
+}
+
+interface AuctionResultsProps {
+  relay: RelayRefetchProp
+  artist: AuctionResults_artist
 }
+
+export const RelayAuctionResults = ContextConsumer(
+  class extends React.Component<Props> {
```

Comment:
This is a pattern I've been following so as to create less intermediate variables which often conflict with export naming, etc

Reply:
Ah, yea, cutting down on intermediate non-exported vars (all of which tended to have generic-y `_Container` or whatever names) is cool 👍